pennylane/qchem/factorization.py

```diff
@@ -270,15 +542,15 @@ def basis_rotation(one_electron, two_electron, tol_factor=1.0e-5):
     >>> core, one, two = qml.qchem.electron_integrals(mol)()
     >>> coeffs, ops, unitaries = basis_rotation(one, two, tol_factor=1.0e-5)
     >>> print(coeffs)
-    [array([ 0.84064649, -2.59579282,  0.84064649,  0.45724992,  0.45724992]),
-     array([ 9.57150297e-05,  5.60006390e-03,  9.57150297e-05,  2.75092558e-03,
-            -9.73801723e-05, -2.79878310e-03, -9.73801723e-05, -2.79878310e-03,
-            -2.79878310e-03, -2.79878310e-03,  2.84747318e-03]),
-     array([ 0.04530262, -0.04530262, -0.04530262, -0.04530262, -0.04530262,
-            0.09060523,  0.04530262]),
-     array([-0.66913628,  1.6874169 , -0.66913628,  0.16584151, -0.68077716,
-            0.16872663, -0.68077716,  0.16872663,  0.16872663,  0.16872663,
-            0.17166195])]
+    [array([-2.59579282,  0.84064649,  0.84064649,  0.45724992,  0.45724992]),
```

Comment:
Values are the same, just the order has rearranged a bit, possibly due to the order in which `.terms()` returns the coefficients and operation pair.

Reply:
OK